edx-platform/pearson-aws-theme/lms/static/sass/partials/lms/theme/_dashboard.scss

```
@@ -1,132 +1,21 @@
 // Dashboard styles.
 
-.view-dashboard div.item {
-  li.course-item {
-    list-style: none;
-
-    .course-container {
-      border: 1px solid $gray;
-      border-radius: 3px;
-    }
-  }
-
-  .items.course.single-course {
-    margin-bottom: 20px;
-  }
-
-  .wrapper-course-image {
-    float: left;
-    margin-right: 2%;
-    width: 23%;
-    max-height: 150px;
-    overflow: hidden;
-
-    .cover {
-      .course-image {
-        width: 100%;
-      }
-    }
-  }
-
-  .course .details .wrapper-course-details {
-    display: flex;
-    flex-flow: row wrap;
-    margin-bottom: 10px;
-    justify-content: space-between;
-
-    .course-title {
-      flex-basis: 100%;
-      padding-top: 10px;
-      font-size: 24px;
-      font-family: $font-family-sans-serif;
-    }
-
-    .course-info [class*="info-"] {
-      font-size: 14px;
-      line-height: 20px;
-    }
-
-    .course-info .info-date-block-container {
-      display: block;
-
-      .info-date-block {
-        color: $secondary;
-      }
-    }
-
-    .wrapper-course-actions {
-      margin-right: 20px;
-    }
-
-    .enter-course {
-      color: $inverse;
-      background: $primary;
-      float: right;
-      line-height: 23px;
-      border: 1px solid $primary;
-      border-radius: 3px;
-      padding: 8px 20px;
+.view-dashboard {
```

```suggestion
.view-dashboard {

  .side-container {
    display: none
  }
  
  @media (min-width: 768px) {
    .main-container {
      margin: 0 auto;
      padding: 40px 0;
      max-width: 1180px;
      min-width: 760px;
      width: 100%;
    }
  }
```

What do you think of adding these changes so it is more similar to the juniper version? Of course if that sidebar is not needed